README.rst

```diff
@@ -13,20 +13,48 @@ lcls_naming_tool
 
 A tool that checks the form and content of names with respect to the LCLS naming convention.
 
+Instructions
+------------
+
+To run the Python script:
+
+1. Set ``$ chmod 775 lcls_naming_tool.py`` to make the file executable.
```

Comment:
This shouldn't be necessary, since git tracks the execute bit of committed files. If you committed this as an executable, it should still be an executable when someone else clones the repo.